linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -1376,39 +1376,37 @@ pub struct ApplicationWrapper<A> {
 }
 
 impl<A> ApplicationWrapper<A> {
-    pub async fn raw_query(&self, query: impl AsRef<str>) -> Result<Value> {
+    pub async fn json_query(&self, query: impl AsRef<str>) -> Result<Value> {
+        let query = query.as_ref();
+        let value = self.raw_query(json!({ "query": query })).await?;
+        Ok(value["data"].clone())
+    }
+
+    pub async fn raw_query<T: Serialize>(&self, query: T) -> Result<Value> {
```

Comment:
_This_ is the JSON one: `run_json_query` or `send_json_query`.

Reply:
Ok, I was unsure about the naming.